### PR TITLE
chore: minor improvements on main page

### PR DESCRIPTION
**Pull request by eddycharly**
## Related issue #

This PR adds minor improvements on main page.

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -43,24 +50,28 @@ security. The Kyverno CLI can be used to test policies and validate resources as
...
+<br/>
+
 # Join our community { class="text-center" }
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/kyverno/kyverno)](https://goreportcard.com/report/github.com/kyverno/kyverno) 
-[![License: Apache-2.0](https://img.shields.io/github/license/kyverno/kyverno?color=blue)](https://github.com/kyverno/kyverno/)
-[![GitHub Repo stars](https://img.shields.io/github/stars/kyverno/kyverno)](https://github.com/kyverno/kyverno/stargazers)
+<br/>
+
+### Interested in learning and contributing ?
```
Comment (chipzoller) [inline content/en/_index.md:59]: Headings shouldn't have punctuation.

Comment (eddycharly) [reply content/en/_index.md:59]: Can you suggest a replacement ?

Comment (chipzoller) [reply content/en/_index.md:59]: Actually, this is fine. No space before punctuation though.

Comment (eddycharly) [reply content/en/_index.md]: Fixed

**Pull request merged**